crates/virtio-queue/src/chain.rs

```diff
@@ -106,15 +106,15 @@ where
         }
 
         // Check the target indirect descriptor table is correctly aligned.
-        if desc.addr().raw_value() & (VIRTQ_DESCRIPTOR_SIZE as u64 - 1) != 0
-            || desc.len() & (VIRTQ_DESCRIPTOR_SIZE as u32 - 1) != 0
+        if desc.addr().raw_value() & (VRING_DESC_ALIGN_SIZE as u64 - 1) != 0
+            || desc.len() & (VRING_DESC_ALIGN_SIZE as u32 - 1) != 0
         {
             return Err(Error::InvalidIndirectDescriptorTable);
         }
 
         // It is safe to do a plain division since we checked above that desc.len() is a multiple of
         // VIRTQ_DESCRIPTOR_SIZE, and VIRTQ_DESCRIPTOR_SIZE is != 0.
```

Comment:
This comment should be updated, VIRTQ_DESCRIPTOR_SIZE was replaced with VRING_DESC_ALIGN_SIZE.